tendermint/src/rpc/endpoint/commit.rs

```diff
@@ -1,6 +1,6 @@
 //! `/commit` endpoint JSONRPC wrapper
 
-use crate::{block, rpc};
+use crate::{amino_types::vote::CanonicalVote, block, lite, rpc, vote::SignedVote, Hash};
```

Comment:
The rpc module currently does not depend on `amino_types`. I think, if possible, we should try to keep it that way. Even if that means introducing another type  under tendermint/block (or similar).